### PR TITLE
Add sentry_debug internal route and RedisIntegration

### PR DESCRIPTION
**Pull request by catastrophe-brandon**
## Link(s) to Jira
- RHCLOUD-7071

## Description of Intent of Change(s)
To help confirm that sentry is working, add an internal route that raises an error.

## Local Testing
How can the feature be exercised? curl the route
How can the bug be exploited and fix confirmed? N/A
Is any special local setup required? N/A

## Checklist
- [ ] if API spec changes are required, is the spec updated?
- [ ] are there any pre/post merge actions required? if so, document here.
- [ ] are theses changes covered by unit tests?
- [ ] if warranted, are documentation changes accounted for?
- [ ] does this require migration changes?
  - [ ] if yes, are they backwards compatible?
- [ ] is there known, direct impact to dependent teams/components?
  -  if yes, how will this be handled?

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Practices Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

**Review by astrozzc — APPROVED** [review]
LGTM

**Review by wcmitchell — APPROVED** [review]
After discussion in dev-meeting, LGTM!

File: rbac/internal/urls.py [hunk 1]
```diff
@@ -16,14 +16,28 @@
...
 from . import views
 
+
+class SentryDiagnosticError(Exception):
+    """Raise this to create an event in Sentry."""
+
+    pass
+
+
+def trigger_error(request):
```
Comment (coderbydesign) [inline rbac/internal/urls.py:31]: Would it make sense to move this into `/rbac/internal/views.py` instead of having it directly in the urls file?

**Review by coderbydesign — COMMENTED** [review]
Tested locally and it looks good! Just had one question on the placement of the view method.

**Comment by catastrophe-brandon** [comment]
Sorry for the delay; I wanted to wait until we've gotten past the issue we hit in production before merging this.

**Pull request merged**